abs.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  build: arrow-cpp
```

Comment:
@skupr-anaconda when arrow-cpp 10.0.1 is uploaded and this is taken out those build issues should go away